### PR TITLE
[CLOSES] Issue#2431 Term Entry: JavaScript Dates .setMilliseconds()

### PR DESCRIPTION
**Pull request by Yashi11**
### Description

A new term entry for JavaScript [Dates](https://www.codecademy.com/resources/docs/javascript/dates) about the .setMilliseconds() method in a new file in docs/content/javascript/concepts/dates/terms/setMilliseconds/setMilliseconds.md with an introductory paragraph about what the method is and does , an example showing the method in use and others as per the requirement.

### Type of Change
- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/dates/terms/setMilliseconds/setMilliseconds.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+dateObject.setMilliseconds(n)
+```
+
+Above code changes millisecond of `dateObject` to n, which is an integer between 1 and 999.
+
+The  return value of `.setMilliseconds()` is the new timestamp after updating milliseconds value of the original Date object in place. The `.setMilliseconds()` method returns the new timestamp value, which represents the number of milliseconds since January 1, 1970 00:00:00 UTC.
+
+## Example
+
+To set the millisecond value of given date to 421 :
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/setMilliseconds/setMilliseconds.md:29]:
```suggestion
To set the millisecond value of given date to 421:
```

File: content/javascript/concepts/dates/terms/setMilliseconds/setMilliseconds.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+This example results in the following output:
+
+```shell
+Original date: 2023-01-01T00:00:00.000Z
+Modified date: 2023-01-01T00:00:00.421Z
+```
+
+## Codebyte Example
+
+The example below is runnable, and first, it defines `eventDate` with given date and time, then the millisecond value of `eventDate` is changed to 112 milliseconds in future. Lastly, a `console.log` call prints the new value of `eventDate` to the screen:
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/setMilliseconds/setMilliseconds.md:48]:
```suggestion
The example below is runnable and demonstrates how to define `eventDate` with a given date and time. It then modifies the millisecond value of `eventDate` to be 112 milliseconds in the future. Finally, a `console.log` call prints the updated value of `eventDate` to the screen:
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
few changes from my side

**Comment by SSwiniarski** [comment]
@Yashi11 tag me when you've addressed @dakshdeepHERE's suggestions and I will do a first maintainer review.

**Comment by Yashi11** [comment]
> @Yashi11 tag me when you've addressed @dakshdeepHERE's suggestions and I will do a first maintainer review.

@SSwiniarski done with the requested changes 👍

File: content/javascript/concepts/dates/terms/setMilliseconds/setMilliseconds.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+The **`.setMilliseconds()`** method sets the milliseconds part of a Date object to desired value. The `.setMilliseconds()` takes an integer argument ranging from 0 to 999, representing the new milliseconds value to be set. Only the milliseconds part of a date and time is modified. 
+
+## Syntax
+
+```pseudo
+dateObject.setMilliseconds(n)
+```
+
+Above code changes millisecond of `dateObject` to n, which is an integer between 1 and 999.
```
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setMilliseconds/setMilliseconds.md:23]:
```suggestion
Above code changes millisecond of `dateObject` to `n`, which is an integer between 1 and 999.
```

File: content/javascript/concepts/dates/terms/setMilliseconds/setMilliseconds.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+```
+
+This example results in the following output:
+
+```shell
+Original date: 2023-01-01T00:00:00.000Z
+Modified date: 2023-01-01T00:00:00.421Z
+```
+
+## Codebyte Example
```
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setMilliseconds/setMilliseconds.md:46]: This codebyte is exactly the same as the example. Can we rewrite it so it's different, at least by using the current date.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Yashi11 I did a review and had two comments.

**Review by caupolicandiaz — APPROVED** [review]
@Yashi11 @SSwiniarski Just pushed a few minor edits. Looks good to merge

**Pull request merged**